looper/cli_pydantic.py

```diff
@@ -122,17 +124,34 @@ def run_looper(args: TopLevelParser, parser: ArgumentParser, test_args=None):
         sys.exit(1)
 
     if subcommand_name == "init":
-        return int(
-            not initiate_looper_config(
-                dotfile_path(),
-                subcommand_args.pep_config,
-                subcommand_args.output_dir,
-                subcommand_args.sample_pipeline_interfaces,
-                subcommand_args.project_pipeline_interfaces,
-                subcommand_args.force_yes,
-            )
+
+        console = Console()
+        console.clear()
+        console.rule(f"\n[magenta]Looper initialization[/magenta]")
+        console.print(
+            "[bold]Would you like to follow a guided tutorial?[/bold]  [green]Y[/green] / [red]n[/red]..."
         )
 
+        selection = None
+        while selection not in ["y", "Y", "n", "N"]:
+            selection = console.input("\nSelection: ")
+
```

Comment:
```suggestion
        selection = None
        while selection not in ["y", "n"]:
            selection = console.input("\nSelection: ").lower().strip()

```

Reply:
I fixed this is another place as well to make it cleaner.